lib/grammers-mtproto/src/transport/abridged.rs

```diff
@@ -112,6 +112,13 @@ impl Transport for Abridged {
     }
 }
 
+impl Tagged for Abridged {
+    fn obfuscated_tag(&mut self) -> [u8; 4] {
+        self.init = true;
```

Comment:
A getter mutating itself doesn't feel right.

Maybe the method should be called `init_tag` or similar, if we really need that. Or otherwise just split the tag step from the init step.

Reply:
I've renamed it to `init_tag`.